### DISCUSSION TITLE
Editor Playfield Selection Should Prioritize Later Objects

### DISCUSSION THREAD

**Discussion by ebolite** [ppy/osu#24354] [Ideas] [answered]
When selecting objects on the playfield, earlier objects are prioritized over later objects, meaning that even objects not currently visible can be accidentally selected if they overlap with what you were actually trying to click. This is especially apparent on slider-heavy maps with large overlaps.

https://github.com/ppy/osu/assets/22662047/d9c1686e-dc9f-4068-81aa-e29871b39727

Either playfield selection should prioritize later hit objects, or objects (specifically slider bodies) which are not on-screen should not be considered for playfield selection.

**Comment by peppy** [comment, marked as answer]
already fixed in next release, see https://github.com/ppy/osu/pull/24289
